src/main/java/org/recordrobotics/charger/commands/manual/ManualClaw.java

```diff
@@ -32,20 +33,23 @@ public void initialize() {
 
 	@Override
 	public void execute() {
+		SmartDashboard.putBoolean("switch state", _claw.getSwitchState());
 		switch (_controls.getClawTurn()) {
 			case OPENING:
-				_claw.brake(false);
 				if(_claw.getSwitchState()){
 					_claw.turn(TURN_SPEED);
 				}else{
 					_claw.turn(0);
 				}
 				break;
 			case GRABING:
-				_claw.brake(true);
+				if (_claw.getPosition() > -0.4) {
```

Comment:
that `-0.4` should really be put into the claw subsystem, so the manual/auto command can be a bit simpler and less duplicated values